public/assets/styles/annexes/mesures.css

```diff
@@ -84,3 +84,12 @@
   background-image: url(../../images/icone_etoile_orange.svg);
   transform: translateY(0.2em);
 }
+
+.mesures-par-statut p.modalites:empty {
+  margin: 0;
+}
+
+.mesures-par-statut p.modalites:not(:empty) {
```

Comment:
Le `:empty` j'adore, mais le `:not(:empty)` je tique un peu. J'ai l'impression que c'est une notation compliquée pour dire « le cas normal ».

Peut-être que si tu remontes cette règle au-dessus de la règle `:empty` alors tu peux enlever le `:not()` ?